firefox_desktop/explores/crashes.explore.lkml

```diff
@@ -0,0 +1,39 @@
+include: "//looker-hub/firefox_desktop/views/crashes_daily.view.lkml"
+include: "../views/crash_usage.view.lkml"
+include: "/shared/views/countries.view.lkml"
+
+
+explore: crash_usage {
+  label: "Crashes"
+  description: "Crash counts for Desktop Firefox, derived from the crash ping."
+
+  join: crashes_daily {
+    sql_on: ${crashes_daily.client_id} = ${crash_usage.client_id}
+            AND ${crashes_daily.submission_date} = ${crash_usage.submission_date} ;;
+    relationship: one_to_one
+    fields: []
+  }
+
+  join: buildhub {
+    sql_on: ${crash_usage.version} = ${buildhub.version} AND ${crash_usage.channel} = ${buildhub.channel} ;;
+    relationship: many_to_one
+    fields: []
```

Comment:
I'm not following these joins, why `fields: []`?

Reply:
We join these views simply to pull in some data for measures or dimensions, but don't actually want to expose any of the fields in them. For example, the buildhub data just gives us the date the build was published, and we use that to calculate a `days_since_published` field when subtracted from `submission_date`.